code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubExternalIdHandler.java

```diff
@@ -77,7 +77,15 @@ public ExternalId untransform(ExternalId externalId) {
         if(StringUtils.isBlank(jwt) && !StringUtils.isBlank(accessToken)){
             try {
                 jwt =  ProjectConstants.AUTH_TYPE + githubTokenHandler.getGithubToken(accessToken).getJwt();
-            } catch (IOException e) {
+            } catch (ClientVisibleException e) {
```

Comment:
No longer deletes the access token. Just doesn't give you a jwt. Which as a result doesn't give you access to any other accounts. eg: Environments.